.github/workflows/docker.yml

```diff
@@ -0,0 +1,71 @@
+# GitHub actions workflow which builds and publishes the docker images.
+
+name: Build docker images
+
+on:
+  push:
+    branches: [ master ]
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  packages: write
+jobs:
+  build:
+    if: github.repository == 'DarkEnergyProcessor/NPPS4'
```

Comment:
Can I change this so it only runs the whole docker build if `DOCKERHUB_USERNAME` and `DOCKERHUB_TOKEN` is present instead of checking the repository?

Reply:
Yes, this should be possible